uitools/toolkitcpp/import/Esri/ArcGISRuntime/Toolkit/PopupView.qml

```diff
@@ -84,6 +85,30 @@ Page {
      */
     property var popup: null
 
+    /*!
+       \brief Boolean that controls if the PopupView will attempt to open URL's with an external browser.
+
+       Defaults to true.
+       \qmlproperty bool openUrlsExternally
+     */
+    property bool openUrlsExternally: true
+
+    /*!
+       \brief Boolean that controls if the PopupView will attempt to open images with an internal viewer.
+
+       Defaults to true.
+       \qmlproperty bool openImagesInternally
+     */
+    property bool openImagesInternally: true
+
+    /*!
+       \brief Boolean that controls if the PopupView will attempt to open attachments with an external viewer.
```

Comment:
perhaps link to the qt open method, which should provide more details about what behavior they can expect